packages/docmanager-extension/src/index.tsx

```diff
@@ -810,6 +822,10 @@ function addCommands(
             type: 'object',
             description: 'The kernel model to use'
           },
+          kernelPreference: {
+            type: 'object',
+            description: 'Override kernel preferences'
```

Comment:
Maybe:

```suggestion
            description: 'Override kernel preferences, see [`IKernelPreference`](https://jupyterlab.readthedocs.io/en/latest/api/interfaces/apputils.ISessionContext.IKernelPreference.html) for possible values'
```

would this render neatly?

Reply:
Just tried the suggestion locally, and it actually seems to be rendering fine:

<img width="2244" height="1076" alt="image" src="https://github.com/user-attachments/assets/246389cf-17ae-46e7-a6e4-d4ad5e07d970" />

So let's go for it, thanks!